report_thesis/src/sections/background.tex

```diff
@@ -205,6 +205,33 @@ \subsubsection{Support Vector Regression (SVR)}
 The optimization problem in \gls{svr} is to find the coefficients $\mathbf{w}$ and $b$ that minimize the norm of $\mathbf{w}$ (i.e., keep the regression function as flat as possible) while ensuring that most data points lie within the $\epsilon$ margin.
 
 \subsubsection{Extra Trees Regressor (ETR)}
+Before discussing the \gls{etr} model, it is important to first understand the concepts of decision trees and \gls{rf}.
+We give an overview of decision trees and \gls{rf} based on \citet{James2023AnIS}.
+Then, we introduce the \gls{etr} model based on \citet{geurtsERF}.
+
+A decision tree is a supervised learning model that partitions data into subsets based on feature values, creating a tree structure.
```

Comment:
I gave a little more thorough explanation of the basics of decision trees when defining gradient boosting. 
Perhaps you could refer back to that? In any case, I think the two sections need to be alinged